gui/src/hooks/imu-logic.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react';
+import { FlatDeviceTracker } from './app';
+import { ImuType } from 'solarxr-protocol';
+
+export function useBnoExists(connectedTrackers: FlatDeviceTracker[]): boolean {
+  const bnoExists = useMemo(
+    () =>
+      connectedTrackers.some(
+        (tracker) =>
+          tracker.tracker.info?.imuType &&
+          [ImuType.BNO055, ImuType.BNO080, ImuType.BNO085].includes(
+            tracker.tracker.info?.imuType
+          )
+      ),
+    [connectedTrackers]
+  );
+
+  return bnoExists;
```

Comment:
Something likely goofs here, as I cannot get the calibration tutorial to trigger with neither of my 3 BNO085 trackers.

Reply:
nice